herbert_core/configuration/@herbert_config/private/process_unit_test_path.m

```diff
@@ -9,6 +9,13 @@
         'herbert_init is not on Matlab search path');
 end
 tests_path = fullfile(rootpath,'_test');
+if ~(exist(test_path,'dir')==7)
```

Comment:
what is this magic `7`?

Reply:
That's Matlab code for directory. file would be 2

Reply:
Matlab is relatively old languge, so has number of such gems lying around.

Reply:
it would be useful to put that in as comment for future record